lib/sift/client.rb

```diff
@@ -1,17 +1,22 @@
 require 'httparty'
 require 'multi_json'
 
+require_relative "./client/decision"
```

Comment:
oh i didn't realize that, I'll ask the `SE`, one thing to note is that our `travis` configs don't test `1.8.7` good catch 👍